app/assets/stylesheets/application.scss

```diff
@@ -12,6 +12,14 @@
  *
  *= require_tree .
  *= require_self
- *= require foundation_and_overrides
 
  */
+
+// Materialize SCSS
+@import "materialize/components/color";
+$primary-color: color("blue", "darken-4") !default;
```

Comment:
Color literals like `blue` should only be used in variable declarations; they should be referred to via variable everywhere else.